serverless.yml

```diff
@@ -0,0 +1,65 @@
+service: bkoninski-aws-workshops-1  # NOTE: update this with your service name
```

Comment:
let's change this bkoninski to <CHANGE_ME> value. I think it's going to be invalid so the accidental deployment would fail.